jep/1/README.adoc

```diff
@@ -640,40 +679,41 @@ This workflow does not dictate specific time frames for any actions.
 and all involved should respond in everyone can agree is timely manner.
 If it becomes necessary to set specific timeframes for action,
-it is the sponsors responsibility to do so.
-Just as the sponsor must build consensus for a JEP,
+it is the Champion's responsibility to do so.
+Just as the Champion must build consensus for a JEP,
 they must also set and communicate a reasonable schedule to keep a JEP moving forward.
 If one or more contributors are not responding
-and the sponsor chooses to move forward without their feedback,
+and the Champion chooses to move forward without their feedback,
 they should document that choice in the "<<Reasoning>>" section of the JEP.
 
+For decisions made by the <<Jenkins Community>>,
+see the timeframes in the link:https://www.jenkins.io/project/governance[Jenkins Governance Document].
+
 ==== Resolving Disputes
 
 Except for decisions by a JEP's <<Reviewer>>,
 the JEP process is run by
 link:https://en.wikipedia.org/wiki/Consensus_decision-making[consensus].
 It is the responsibility of every contributor to respect other contributors,
 listen to their perspectives, and attempt to find solutions that work for everyone.
+The link:https://www.jenkins.io/project/conduct/[Jenkins Code of Conduct] applies
+to all sides and to all aspects of the JEP process.
 
 If consensus cannot be achieved on a JEP,
 contributors may request that the <<reviewer, JEP Reviewer>> intervene.
 The reviewer will consider the matter, and render their decision,
 including describing what actions will be taken and documenting their reasoning.
 
-For disputes involving a decision by a <<BDFL Delegate>>
-or the overall JEP process (rather than one specific JEP),
-contributors may request that the <<BDFL>> intervene.
-The BDFL will consider the matter and render their decision,
-including describing what actions will be taken and documenting their reasoning.
-The BDFL's decision may include technical direction and other specific instructions as needed.
-
-If contributors believe a decision made by the BDFL runs counter to the best interests to Jenkins project,
-they may request the Governance meeting review the BDFL's decision.
-The Governance meeting will take up the matter and render a decision within a reasonable timeframe.
-Similar to the judiciary, the Governance meeting will _not_ make technical decisions,
-they will only affirm or reject the BDFL's decision.
+If contributors believe a decision made by the <<Reviewer>> runs counter to the best interests to Jenkins project,
+they may request the <<Jenkins community>> to review the decision.
+Also, they may do in the case of the  the overall JEP process disputes (rather than one specific JEP).
+The community or, if needed, the Jenkins Governance Board,
+will take up the matter and render a decision within a reasonable timeframe.
+Similar to the judiciary, the community will _not_ make technical decisions,
+they will only affirm or reject the Reviewer's decision.
```

Comment:
This part of the process needs more rewriting.
Not sure how to phrase it yet though.